src/components/Filters/Indicators.tsx

```diff
@@ -1,9 +1,26 @@
-import React, { Component } from 'react';
-import { Select } from 'antd';
-import { STATS, styles } from '../../utils/constants.js';
+import React, {Component} from 'react';
+import {Select} from 'antd';
+import {STATS, styles} from '../../utils/constants';
+import {SelectValue} from "antd/lib/select";
```

Comment:
This is a specific `SelectValue` type? How did you know to do this?

Reply:
Some popular libraries will come with types 

Take a look at 
`node_modules/antd/lib/select/index.d.ts`